flink-tests/src/test/java/org/apache/flink/test/recovery/AbstractTaskManagerProcessFailureRecoveryTest.java

```diff
@@ -188,6 +196,14 @@ public void run() {
             taskManagerProcess1.destroy();
             waitForShutdown("TaskManager 1", taskManagerProcess1);
 
+            // TODO do a proper check here
+            assertTrue(
+                    Paths.get(taskManagerWorkingDir.getAbsolutePath(), "allocations")
+                                    .toFile()
+                                    .list()
+                                    .length
+                            != 0);
```

Comment:
I would suggest to add a dedicated test instead of piggy-backing on some other test.